handler/src/main/java/io/netty/handler/ssl/Java8SslUtils.java

```diff
@@ -16,15 +16,18 @@
 package io.netty.handler.ssl;
 
 import javax.net.ssl.SNIHostName;
+//import javax.net.ssl.SNIMatcher;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)